qe/plugins/org.jboss.tools.fuse.qe.reddeer/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,37 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %Bundle-Name
+Bundle-SymbolicName: org.jboss.tools.fuse.qe.reddeer;singleton:=true
+Bundle-Version: 10.1.0.qualifier
+Bundle-Activator: org.jboss.tools.fuse.qe.reddeer.Activator
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Require-Bundle: org.eclipse.ui,
+ org.eclipse.ui.ide,
+ org.eclipse.gef,
+ org.eclipse.osgi,
+ org.eclipse.core.resources,
+ org.eclipse.core.runtime,
+ org.eclipse.graphiti.ui,
+ com.jcraft.jsch;bundle-version="0.1.50",
+ org.eclipse.wst.server.core;bundle-version="1.10.0",
+ org.hamcrest.core;bundle-version="1.3.0",
+ org.eclipse.reddeer.go;bundle-version="2.0.0",
+ org.eclipse.reddeer.gef;bundle-version="2.0.0",
+ org.eclipse.reddeer.graphiti;bundle-version="2.0.0"
+Export-Package: org.jboss.tools.fuse.qe.reddeer,
+ org.jboss.tools.fuse.qe.reddeer.component,
+ org.jboss.tools.fuse.qe.reddeer.condition,
+ org.jboss.tools.fuse.qe.reddeer.debug,
+ org.jboss.tools.fuse.qe.reddeer.editor,
+ org.jboss.tools.fuse.qe.reddeer.perspectives,
+ org.jboss.tools.fuse.qe.reddeer.preference,
+ org.jboss.tools.fuse.qe.reddeer.projectexplorer,
+ org.jboss.tools.fuse.qe.reddeer.requirement,
+ org.jboss.tools.fuse.qe.reddeer.runtime,
+ org.jboss.tools.fuse.qe.reddeer.runtime.impl,
+ org.jboss.tools.fuse.qe.reddeer.utils,
+ org.jboss.tools.fuse.qe.reddeer.view,
+ org.jboss.tools.fuse.qe.reddeer.wizard
+Bundle-Vendor: %Bundle-Vendor
+Import-Package: com.fasterxml.jackson.annotation
+Eclipse-BundleShape: dir
```

Comment:
Out of curiosity, why is it BundleShape dir?

Reply:
Because we copy some files from that bundle and it is easier to copy from dir than from jar